lib/plugins/deploy.js

```diff
@@ -56,6 +59,16 @@ class Deploy {
           return;
         }
 
+        if (this.serverless.service.provider.name === 'aws') {
```

Comment:
Maybe it would be better to put it into `lib/plugins/aws/deploy.js`, then this conditional wouldn't be needed

Reply:
I thought of that first, but unfortunately, `package` is executed before we even get to `aws:deploy:deploy`, that's why I had to move it here, I'll see if maybe we can do something else about it but I'm not sure if it's going to be possible

Reply:
hmm.. this is part of `before:deploy:deploy` hook here. Do you mean that putting this logic in very same hook in context of `lib/plugins/aws/deploy.js` will produce different result?

Reply:
Yes, exactly, if I put that logic in `before:deploy:deploy` hook of `aws/deploy` then the packaging spinner will appear for some time and only after that the line will be printed

Reply:
Ok, so goal is to show this message before packaging? Technically right at the begining of deploy command operation right?

Then I think cleanest way is to set this up in `initialize` hook in `lib/plugins/aws/deploy.js` after confirming that we deal with `deploy` command

Reply:
Changed :+1: